ios/Capacitor/Capacitor/CAPUNUserNotificationCenterDelegate.swift

```diff
@@ -133,7 +131,9 @@ public class CAPUNUserNotificationCenterDelegate : NSObject, UNUserNotificationC
       "title": content.title,
       "subtitle": content.subtitle,
       "body": content.body,
-      "badge": content.badge ?? 1
+      "badge": content.badge ?? 1,
+      "data": content.userInfo["custom_data"],
```

Comment:
I think it should be better to just pass the whole userInfo as data instead of forcing to use "custom_data" key as that is not a standard key and we don't want to force people to modify their payload to match this.

Reply:
We're using FCM to send iOS notifications. custom_data is standard there but I can see now that it's not a standard iOS attribute.

Reply:
Interesting, are you using some plugin? Or you just integrated it into your project?

Reply:
APN tokens can be registered with FCM so that we can have a single API for sending messages. There's no ionic/capacitor plugin. It's just the format that FCM sends the data.